src/change/direct.rs

```diff
@@ -0,0 +1,84 @@
+use crate::dtls::Fingerprint;
+use crate::ice::IceCreds;
+use crate::rtp::Direction;
+use crate::rtp::Mid;
+use crate::Rtc;
+use crate::RtcError;
+
+/// Direct change strategy.
+///
+/// Makes immediate changes to the Rtc session without any Sdp OFFER/ANSWER.
+pub struct DirectApi<'a> {
+    rtc: &'a mut Rtc,
+}
+
+impl<'a> DirectApi<'a> {
+    /// Creates a new instance of the `DirectApi` struct with the specified `Rtc` instance.
+    ///
+    /// The `DirectApi` struct provides a high-level API for interacting with a WebRTC peer connection,
+    /// and the `Rtc` instance provides low-level access to the underlying WebRTC functionality.
+    pub fn new(rtc: &'a mut Rtc) -> Self {
+        DirectApi { rtc }
+    }
+
+    /// Sets the ICE controlling flag for this peer connection.
+    ///
+    /// If `controlling` is `true`, this peer connection is set as the ICE controlling agent,
+    /// meaning it will take the initiative to send connectivity checks and control the pace of
+    /// connectivity checks sent between two peers during the ICE session.
+    ///
+    /// If `controlling` is `false`, this peer connection is set as the ICE controlled agent,
+    /// meaning it will respond to connectivity checks sent by the controlling agent.
+    pub fn ice_controlling(&mut self, controlling: bool) {
```

Comment:
Because this API mixes getter/setters, I want to make it clear what is what. Currently `local_ice_credentials()` and `remote_ice_credentials()` sounds like they are the same, but one is a getter and one is a setter. I think I want a `set_xxx` prefix on the setters. 

What do you think?